src/Media.Plugin.Android/MediaImplementation.cs

```diff
@@ -86,7 +86,8 @@ public MediaImplementation()
             {
                 try
                 {
-                    await FixOrientationAndResizeAsync(media.Path, options.PhotoSize, options.CompressionQuality, options.CustomPhotoSize);
+                    //await FixOrientationAndResizeAsync(media.Path, options.PhotoSize, options.CompressionQuality, options.CustomPhotoSize);
```

Comment:
Is there a reason you are no longer fixing orientation? This is the most requested feature to auto rotate the image. This will only resize it.